src/interaction/index.ts

```diff
@@ -1,3 +1,4 @@
+import * as change from "@src/interaction/change";
```

Comment:
Will this enable input events?

Reply:
Yes, it was a miss from many version before. It wires it up now.